packages/admin/admin/src/mui/menu/Item.styles.ts

```diff
@@ -0,0 +1,177 @@
+import { Theme } from "@mui/material";
+import { createStyles } from "@mui/styles";
+
+import { MenuItemProps, MuiListItemProps } from "./Item";
+
+export type MenuItemClassKey = "root" | "level1" | "level2" | "level3" | "hasIcon" | "hasSecondaryText" | "hasSecondaryAction";
+
+const colors = {
+    textLevel1: "#242424",
+    textLevel2: "#17181A",
+};
+
+export const styles = (theme: Theme) =>
+    createStyles<MenuItemClassKey, MenuItemProps & MuiListItemProps>({
+        root: {
+            flexShrink: 0,
+            "&:after": {
+                content: "''",
+                position: "absolute",
+                top: 0,
+                right: 0,
+                bottom: 0,
+                width: 2,
+            },
+            "& [class*='MuiListItemIcon-root']": {
+                color: colors.textLevel1,
+                minWidth: 28,
+            },
+            "& [class*='MuiListItemText-inset']": {
+                paddingLeft: 28,
+            },
+            "& [class*='MuiSvgIcon-root']": {
+                fontSize: 16,
+            },
+        },
+        level1: {
+            borderBottom: `1px solid ${theme.palette.grey[50]}`,
+            boxSizing: "border-box",
+            color: colors.textLevel1,
+            paddingLeft: 20,
+            paddingRight: 20,
+            paddingTop: 16,
+            paddingBottom: 16,
+            "&[class*='Mui-selected']": {
+                backgroundColor: theme.palette.grey[50],
+                color: theme.palette.primary.main,
+                "&:after": {
+                    backgroundColor: theme.palette.primary.main,
+                },
+                "& [class*='MuiListItemIcon-root']": {
+                    color: theme.palette.primary.main,
+                },
+            },
+            "& [class*='MuiListItemText-primary']": {
+                fontWeight: theme.typography.fontWeightMedium,
+                fontSize: 16,
+                lineHeight: "20px",
+            },
+        },
+        level2: {
+            color: colors.textLevel2,
+            paddingLeft: 20,
+            paddingRight: 20,
+            paddingTop: 10,
+            paddingBottom: 10,
+            "&:last-child": {
+                borderBottom: ({ level, hasChildElements, isCollapsibleOpen }) =>
+                    level === 2 && (!hasChildElements || !isCollapsibleOpen) ? `1px solid ${theme.palette.grey[50]}` : "initial",
```

Comment:
For what do you need this?

Reply:
To have the border move from this:
<img width="299" alt="Screenshot 2023-12-22 at 09 37 57" src="https://github.com/vivid-planet/comet/assets/56400587/5c723a78-9975-4b59-90a0-bdc00b5c77e3">
to this:
<img width="284" alt="Screenshot 2023-12-22 at 09 38 06" src="https://github.com/vivid-planet/comet/assets/56400587/b2c40cf7-c86c-4d9e-9641-87253cc4d504">